src/glyphs.js

```diff
@@ -273,4 +351,26 @@ router.post('/:_id/messages/:num', (req, res) => {
   }
 });
 
+// This route is used to import messages from a spreadsheet (CSV)
+router.use(formidable()).post('/import/', (req, res) => {
+  const processed = processImportRequest(req);
+  let response = [];
+
+  if (processed.errors.length > 0) {
+    res.json({
+      'errors': processed.errors
+    });
+  } else {
+    if (req.fields.type === "overwrite") overwriteMessages(req.files[""].path);
+
+    let first = true;
```

Comment:
Am I mistaken or will this import messages after already overwriting them?  Is there any check that prevents that?

Reply:
Rename `overwriteMessages`, and that will go a long way in making this make sense.